doc/coq.rst

```diff
@@ -0,0 +1,507 @@
+.. _coq:
+
+***
+Coq
+***
+
+
+Dune is able to build Coq projects and plugins with support for source file
+preprocessing, extraction, and documenation.
+
+
+.. contents:: Table of Contents
+    :depth: 3
+
+.. _coq-theory:
+
+coq.theory
+----------
+
+A *Coq theory* is a collection of ``.v`` files. Together with OCaml libraries
+linking to the Coq API, a Coq theory is termed a *Coq plugin*. Both may also be
+termed a *Coq Project*. To enable Coq support in a Dune project. The language
```

Comment:
Maybe "to enable to Coq Dune language"? I'm not sure what the right terms are here.